README.md

```diff
@@ -1,12 +1,24 @@
-# Authors
-- Anssi Kostiainen (Intel)
-- Belem Zhang (Intel)
-- Wei Wang (Intel)
-
 # Local Peer-to-Peer API
 
-For the latest proposals, see the [Local Peer-to-Peer API Explainer](EXPLAINER.md).
+[Local Peer-to-Peer](https://WICG.github.io/local-peer-to-peer/) is a Web platform API proposal for local communication between browsers without the aid of a server.
```

Comment:
Using https://github.com/WICG/local-peer-to-peer instead of https://WICG.github.io/local-peer-to-peer/?
https://WICG.github.io/local-peer-to-peer/ gets 404 error.

Reply:
Yes, once this is merged to the upstream repo, we can setup the hosting of the spec using Github Pages. In that case, path wil host the rendered specification itself.